code/modules/clothing/suits/bio.dm

```diff
@@ -47,10 +47,32 @@
 
 //Security biosuit, grey with red stripe across the chest
 /obj/item/clothing/head/bio_hood/security
+	name = "security bio hood"
+	desc = "A hood that protects the head and face from biological comtaminants. It has a reinforced synthetic lining to protect against tearing."
 	icon_state = "bio_security"
+	armor = list(melee = 25, bullet = 10, laser = 15, energy = 5, bomb = 5, bio = 100, rad = 20)
 
 /obj/item/clothing/suit/bio_suit/security
+	name = "security bio suit"
+	desc = "A suit that protects against biological contamination. It has a reinforced synthetic lining to protect against tearing."
 	icon_state = "bio_security"
+	allowed = list(
+		/obj/item/weapon/gun/energy,
+		/obj/item/weapon/reagent_containers/spray/pepper,
+		/obj/item/weapon/gun/projectile,
+		/obj/item/ammo_storage,
+		/obj/item/ammo_casing,
+		/obj/item/weapon/melee/baton,
+		/obj/item/weapon/handcuffs,
+		/obj/item/weapon/gun/lawgiver,
```

Comment:
why not just any /obj/item/weapon/gun?

Reply:
Because I was just copying the list from /obj/item/clothing/suit/armor for consistency's sake.

Reply:
Yes, copypaste makes our code more consistent and easier to maintain.

Reply:
Clearly someone thought that security armour should not use weapon/gun and they may have had good reason to do so. This is not the PR to change that. If you'd rather this PR be titled "a complete overhaul of our exosuits" then that is on you.